### PR TITLE
Pause menu

### PR DESCRIPTION
**Pull request by rster2002**
## Changes
Adds the code for controlling a pause menu. For now it's used to quickly switch scenes.

### CODE REVIEW

File: Demo/Demo/Assets/Scripts/MobileOnly.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ void Start()
         bool isMobile = Application.platform == RuntimePlatform.Android ||
                         Application.platform == RuntimePlatform.IPhonePlayer;
 
+        isMobile = true;
```
Comment (suchmememanyskill) [inline Demo/Demo/Assets/Scripts/MobileOnly.cs:14]: Is this supposed to be there?

**Review by suchmememanyskill — CHANGES_REQUESTED** [review]

File: Demo/Demo/Assets/Scripts/MobileOnly.cs (hunk 1 above)
Comment (rster2002) [reply Demo/Demo/Assets/Scripts/MobileOnly.cs]: Whoops

**Comment by rster2002** [comment]
For some unknown reason level 1 is still screwed

**Review by suchmememanyskill — APPROVED** [review]

**Comment by suchmememanyskill** [comment]
Please create an issue for the level 1 thing as soon as this is merged

**Comment by rster2002** [comment]
Yes will do, but do we need level 1 for anything other than the tutorial?

**Comment by rster2002** [comment]
Level 1 issue: https://github.com/OwenVanRijn/vigilant-umbrella/issues/22

**Comment by suchmememanyskill** [comment]
I'm gonna assume this should be merged

No we don't need level 1 at all anymore, really. At least until level 2 becomes a lot better probs worth to keep it in

**Pull request merged**